controllers/awsmachine_controller.go

```diff
@@ -588,10 +600,12 @@ func (r *AWSMachineReconciler) reconcileLBAttachment(machineScope *scope.Machine
 	if err := elbsvc.RegisterInstanceWithAPIServerELB(i); err != nil {
```

Comment:
Would it also makes sense to update the condition above after calling `elbsvc.DeregisterInstanceFromAPIServerELB()`?

Reply:
I think it makes sense for when the Deregister fails. With deletion / clean up flows, I see conditions being useful when things go wrong or get stuck. Will update